src/Symfony/Component/Lock/Tests/LockTest.php

```diff
@@ -50,22 +50,6 @@ public function testAcquireNoBlockingStoreInterface()
         $this->assertTrue($lock->acquire(false));
     }
 
-    /**
-     * @group legacy
-     */
-    public function testPassingOldStoreInterface()
-    {
-        $key = new Key(uniqid(__METHOD__, true));
-        $store = $this->getMockBuilder(PersistingStoreInterface::class)->getMock();
```

Comment:
on 4.4 this should have been mocking `StoreInterface::class` instead

Reply:
see https://github.com/symfony/symfony/pull/32938/files